### ISSUE TITLE
Make the wrong namespace warnings autocorrect in Migration/DepartmentName

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#7815]
## Is your feature request related to a problem? Please describe.

It would be really great if all the 'wrong namespace' warnings that get thrown during RuboCop runs would be detected / corrected with Migration/DepartmentName.

## Describe the solution you'd like

The current warnign could stay, but the Migration/DepartmentName cop would also detect and correct all these warnings so that users could easily migrate to the new names and cleanup their old rubocop usage.

Example warnings:
```
/Users/tsmith/dev/the_world2/cisco-cookbook/files/default/vendor/gems/cisco_node_utils-1.7.0/lib/cisco_node_utils/ace.rb: Style/AccessorMethodName has the wrong namespace - should be Naming
/Users/tsmith/dev/the_world2/cisco-cookbook/files/default/vendor/gems/cisco_node_utils-1.7.0/lib/cisco_node_utils/ace.rb: Metrics/LineLength has the wrong namespace - should be Layout
/Users/tsmith/dev/the_world2/cisco-cookbook/files/default/vendor/gems/cisco_node_utils-1.7.0/lib/cisco_node_utils/ace.rb: Metrics/LineLength has the wrong namespace - should be Layout
```

## Describe alternatives you've considered

Nothing really.

**Comment by koic** [comment]
Thank you for the feedback. These migrations are currently provided by Mry gem, not by RuboCop itself. Mry gem helps to migrate those departments.
https://github.com/pocke/mry
